src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+
+const INITIAL_FACILITY_DISPLAY_COUNT = 5;
```

Comment:
Storing initial display count in a const as suggested and using it throughout this component.